modules/ROOT/pages/index.adoc

```diff
@@ -8,26 +15,96 @@
 
 :console: 0
 
-This is the Neo4j Operations Manual, which includes all the operational details and instructions for installing and deploying Neo4j on-premise and in the cloud.
+[.display.cards.selectable]
 
-For all information on *upgrading and migrating Neo4j*, see the link:{neo4j-docs-base-uri}/upgrade-migration-guide/current/[Neo4j Upgrade and Migration Guide].
+== Manage and monitor your deployments
 
-For more information on link:https://neo4j.com/aura/[Aura], the Neo4j fully managed cloud service, see the link:{neo4j-docs-base-uri}/aura[Neo4j Aura Manual].
+Find here details and instructions for installing and deploying Neo4j on-premises and in the cloud.
+Follow tutorials and find administrative commands to help you manage and monitor your projects.
+The latest version of Neo4j is *{neo4j-version-exact}*.
 
+[.widget.lists]
+== Tailored for you
```

Comment:
```suggestion
```

Reply:
This doesn't show in the page. It just needs to be there for the component to work properly.

Reply:
Ok. That's fine. We need to revert the other heading then.